modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/CKEditor5ClassicEditorConfigContributor.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.frontend.editor.ckeditor.web.internal.editor.configuration;
+
+import com.liferay.portal.kernel.editor.configuration.BaseEditorConfigContributor;
+import com.liferay.portal.kernel.editor.configuration.EditorConfigContributor;
+import com.liferay.portal.kernel.json.JSONObject;
+import com.liferay.portal.kernel.portlet.RequestBackedPortletURLFactory;
+import com.liferay.portal.kernel.theme.ThemeDisplay;
+import com.liferay.portal.kernel.util.GetterUtil;
+
+import java.util.Map;
+
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * @author Marko Cikos
+ */
+@Component(
+	property = "editor.name=ckeditor5_classic",
+	service = EditorConfigContributor.class
+)
+public class CKEditor5ClassicEditorConfigContributor
+	extends BaseEditorConfigContributor {
+
+	@Override
+	public void populateConfigJSONObject(
+		JSONObject jsonObject, Map<String, Object> inputEditorTaglibAttributes,
+		ThemeDisplay themeDisplay,
+		RequestBackedPortletURLFactory requestBackedPortletURLFactory) {
+
+		String namespace = GetterUtil.getString(
+			inputEditorTaglibAttributes.get(
+				"liferay-ui:input-editor:namespace"));
+		String name = GetterUtil.getString(
+			inputEditorTaglibAttributes.get("liferay-ui:input-editor:name"));
+
+		jsonObject.put(
+			"itemSelectorEventName", namespace + name + "selectItem"
+		).put(
+			"preset", "advanced"
+		);
```

Comment:
This file is suppose to be the base config contributor for CKE5 classic version. Note that it is not in the sample module. We are being opinionated here, setting "advanced" preset as the default one. The preset includes image selector, so it requires `itemSelectorEventName`.

In the sample config contributor, there is still [setting of the placeholder](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-sample-web/src/main/java/com/liferay/frontend/editor/ckeditor/sample/web/internal/editor/configuration/CKEditor5ClassicEditorConfigContributor.java#L38-L40), and this is still checked in a test.